### PR TITLE
inci

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by melo0712 — APPROVED** [review]
It looks awesome

File: src/test/java/com/trycloud/tests/US003_TC001_inci.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+public class US003_TC001_inci extends TestBase {
+    @Test(priority = 1,description = "verify users can access to Files module ")
+    public void test1(){
+        //1.log in as a user
+        login();
+        //2.open Files module
+        FilesPage filesPage=new FilesPage();
+        filesPage.clickFilesButton();
+        BrowserUtils.sleep(3);
```
Comment (mehmetdurkan) [inline src/test/java/com/trycloud/tests/US003_TC001_inci.java:21]: Try to use dynamic waits

File: src/test/java/com/trycloud/tests/US003_TC001_inci.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        Assert.assertTrue(Driver.getDriver().getTitle().contains("Files"));
+    }
+   @AfterClass
+    public void teardown() {
```
Comment (mehmetdurkan) [inline src/test/java/com/trycloud/tests/US003_TC001_inci.java:25]: I think this should be inherited from parent as all test classes will need it

File: src/test/java/com/trycloud/tests/base/US003_TC002_inci.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        FilesPage filesPage=new FilesPage();
+        filesPage.clickFilesButton();
+        BrowserUtils.sleep(3);
+      //3.click the top lef checkbox of the table
+       filesPage.selectAllFilesButton.click();
+      //4.Assert all the files are selected
+       //I will use table to assert all check boxes are displayed
+    }
+    @AfterClass
+    public void teardown() {
```
Comment (mehmetdurkan) [inline src/test/java/com/trycloud/tests/base/US003_TC002_inci.java:27]: This should be coming from TestBase

File: trycloudautomation.iml [hunk 4]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mehmetdurkan) [inline trycloudautomation.iml:1]: Lets add this to gitignore and dont check iml files in release anymore

**Pull request merged**